tonic/src/lib.rs

```diff
@@ -30,9 +30,9 @@
 //! - `tls-webpki-roots`: Add the standard trust roots from the `webpki-roots` crate to
 //! `rustls`-based gRPC clients. Not enabled by default.
 //! - `prost`: Enables the [`prost`] based gRPC [`Codec`] implementation.
-//! - `compression`: Enables compressing requests, responses, and streams. Note
-//! that you must enable the `compression` feature on both `tonic` and
-//! `tonic-build` to use it. Depends on [flate2]. Not enabled by default.
+//! - `gzip`: Enables compressing requests, responses, and streams.
+//! Depends on [flate2]. Not enabled by default.
+//! Replaces the `compression` flag from earlier versions of `tonic`.
```

Comment:
Probably should include <=0.7 to specify which version?
```suggestion
//! Replaces the `compression` flag from earlier versions of `tonic` (<= 0.7).
```